modules/core/02-client/types/events.go

```diff
@@ -21,14 +21,15 @@ const (
 
 // IBC client events vars
 var (
-	EventTypeCreateClient          = "create_client"
-	EventTypeUpdateClient          = "update_client"
-	EventTypeUpgradeClient         = "upgrade_client"
-	EventTypeSubmitMisbehaviour    = "client_misbehaviour"
-	EventTypeUpdateClientProposal  = "update_client_proposal"
-	EventTypeUpgradeChain          = "upgrade_chain"
-	EventTypeUpgradeClientProposal = "upgrade_client_proposal"
-	EventTypeRecoverClient         = "recover_client"
+	EventTypeCreateClient               = "create_client"
+	EventTypeUpdateClient               = "update_client"
+	EventTypeUpgradeClient              = "upgrade_client"
+	EventTypeSubmitMisbehaviour         = "client_misbehaviour"
+	EventTypeUpdateClientProposal       = "update_client_proposal"
+	EventTypeUpgradeChain               = "upgrade_chain"
+	EventTypeUpgradeClientProposal      = "upgrade_client_proposal"
+	EventTypeScheduleIBCSoftwareUpgrade = "schedule_ibc_client_upgrade"
```

Comment:
should the value be `schedule_ibc_software_upgrade`?

Reply:
i thought about this, but actually we are scheduling an upgrade of a client state. but happy to change it to `schedule_ibc_software_upgrade` for consistency with the method name, if that makes more sense!